internal/mptrie/store/update_and_query.go

```diff
@@ -171,6 +171,9 @@ func (s *Store) CommitChanges(blockNum uint64) error {
 	if err := s.trieDataDB.Write(batch, &opt.WriteOptions{Sync: true}); err != nil {
 		return err
 	}
+	s.logger.Infof("mptrie commit - total get ops %d, db get ops %d, put ops %d, persist ops %d", nodeGetTotal, nodeGetDB, len(s.inMemoryNodes), len(s.nodesToPersist))
+	nodeGetTotal = 0
+	nodeGetDB = 0
```

Comment:
Please remove. You can add time taken to persistTrie but for low level metrics, let's not use logs.

Reply:
ok